README.md

```diff
@@ -47,6 +48,17 @@
 
 - **[add score]**
 - **[refresh score]**
+- **[unique design]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://memicelis.github.io/Leaderboard/dist)
```

Comment:
Also, your functionality is not working on the live demo link, but when I run project locally everything looks great.